data/templates/load-balancing/haproxy.cfg.j2

```diff
@@ -82,8 +82,8 @@ frontend {{ front }}
 {%         endif %}
 {%         set ssl_directive = 'ssl' if ssl_front else '' %}
 {%         if front_config.listen_address is vyos_defined %}
-{%             for address in front_config.listen_address %}
-    bind {{ address | bracketize_ipv6 }}:{{ front_config.port }} {{ ssl_directive }} {{ ssl_front | join(' ') }}
+{%             for address, server_config in front_config.listen_address.items() %}
```

Comment:
Is there a real use case when we need different options for the same frontend/service?
```
set load-balancing haproxy service foo listen-address 192.0.2.1
set load-balancing haproxy service foo listen-address 192.0.2.2 proxy
```
VS
```
set load-balancing haproxy service foo listen-address 192.0.2.1
set load-balancing haproxy service foo listen-address 192.0.2.2
set load-balancing haproxy service foo accept-proxy
```
And use proxy for each bind address of the frontend

Reply:
@nobiit thanks!